ptz-explainer.md

```diff
@@ -0,0 +1,166 @@
+# Explainer: Pan/Tilt support for Camera
+
+## Authors:
+
+- fbeaufort@google.com
+
+## Participate
+
+- https://github.com/w3c/mediacapture-image/issues
+
+## Introduction
+
+[Some cameras] have the ability to pan and tilt which is especially useful in
+video conferencing (e.g. steering the camera to face the speaker).
```

Comment:
nit: Remove the (steering the camera) as this is in the Goals section.

Reply:
Done.